linkerd/app/core/src/errors/respond.rs

```diff
@@ -385,7 +385,12 @@ where
         };
 
         let rsp = info_span!("rescue", client.addr = %self.client_addr()).in_scope(|| {
-            tracing::info!(error, "Request failed");
+            if self.is_grpc {
```

Comment:
Looks like the log lines indicate we want to check if this is _not_ gRPC:

```suggestion
            if !self.is_grpc {
```

Reply:
Thank you!